src/exx_kernel_default.f90

```diff
@@ -1173,36 +1151,25 @@ subroutine m_kern_exx_cri(k_off, kpart, ib_nd_acc, ibaddr, nbnab, &
                    !if ( screen_ij < range_ij ) then
                    !write(*,*) 'j',j, 'global_num',jb%global_num,'spe',jb%spec
                    !
-                   if ( exx_alloc ) call exx_mem_alloc(extent,kg%nsup,jb%nsup,'rho_kj','alloc')
                    if ( exx_alloc ) call exx_mem_alloc(extent,jb%nsup,0,'phi_j','alloc')
                    !
                    call exx_phi_on_grid(inode,jb%global_num,jb%spec,extent, &
                         jb%xyz,jb%nsup,phi_j,r_int,xyz_zero)             
                    !
-                   call start_timer(tmr_std_exx_accumul)
-                   rho_kj = zero
-                   do nsf1 = 1, kg%nsup                         
-                      do nsf2 = 1, jb%nsup
-                         !
-                         rho_kj(:,:,:,nsf1,nsf2) = &
-                              Phy_k(:,:,:,nsf1) * phi_j(:,:,:,nsf2)
-                         !
-                      end do
-                   end do
-                   !
-                   call stop_timer(tmr_std_exx_accumul,.true.)
-                   !
-                   if ( exx_alloc ) call exx_mem_alloc(extent,jb%nsup,0,'phi_j','dealloc')                   
-                   if ( exx_alloc ) call exx_mem_alloc(extent,kg%nsup,jb%nsup,'vhf_kj','alloc')                   
+                   if ( exx_alloc ) call exx_mem_alloc(extent,0,0,'Ome_kj','alloc')
                    !
+                   call start_timer(tmr_std_exx_accumul)
+                   !$omp parallel do schedule(runtime) collapse(2) default(none) reduction(+: c) &
+                   !$omp     shared(kg,jb,tmr_std_exx_poisson,tmr_std_exx_accumul,Phy_k,phi_j,phi_k,ncbeg,ia,tmr_std_exx_matmult,ewald_pot,phi_i, &
+                   !$omp            exx_psolver,exx_pscheme,extent,dv,ewald_rho,inode,pulay_radius,p_omega,p_gauss,w_gauss,reckernel_3d,r_int) &
+                   !$omp     private(nsf1,nsf2,work_out_3d,work_in_3d,ewald_charge,Ome_kj, &
+                   !$omp             ncaddr,nsf3,exx_mat_elem,r,s,t)

```

Comment:
Could you shorten the `shared()` lines? Sticking to the 132 character limit would be good style. Otherwise we need to start enforcing the no line length limit in the compiler flags.